demo/src/screens/componentScreens/DialogScreen.js

```diff
@@ -91,9 +104,21 @@ class DialogScreen extends Component {
           centerH
           onDismiss={() => this.setState({showDialog5: false})}
           animationConfig={{duration: 1000}}
```

Comment:
let's remove the animationConfig here so the example will look better

Reply:
done